lib/auth0/api/v2/tickets.rb

```diff
@@ -29,22 +29,40 @@ def post_email_verification(user_id, result_url: nil, ttl_sec: nil)
 
         # Create a password change ticket
         # @see https://auth0.com/docs/api/v2#!/Tickets/post_password_change
-        # @param new_password [string] The password to be set for the user once the ticket is used.
-        # @param user_id [string] The user_id of for which the ticket is to be created.
         # @param result_url [string] The user will be redirected to this endpoint once the ticket is used.
+        # @param user_id [string] The user_id of for which the ticket is to be created.
         # @param connection_id [string] The connection that provides the identity for which the password is to be
         # changed. If sending this parameter, the email is also required and the user_id is invalid.
         # @param email [string] The user's email.
+        # @param ttl_sec [integer] The ticket's lifetime in seconds starting from the moment of creation. After
+        # expiration, the ticket cannot be used to change the user's password. If not specified or if you send 0,
+        # the Auth0 default lifetime of 5 days will be applied.
+        # @param mark_email_as_verified [boolean] true if email_verified attribute must be set to true once password is
+        # changed, false if email_verified attribute should not be updated
+        # @param includeEmailInRedirect [boolean] Whether or not we include the email as part of the returnUrl
+        # in the reset_email
+        # @param new_password [string] The password to be set for the user once the ticket is used.
         #
         # @return [json] Returns the created ticket url.
-        def post_password_change(new_password: nil, user_id: nil, result_url: nil, connection_id: nil, email: nil)
+        def post_password_change(
+            result_url: nil,
+            user_id: nil,
+            connection_id: nil,
+            email: nil,
+            ttl_sec: nil,
+            mark_email_as_verified: nil,
+            includeEmailInRedirect: nil,
+            new_password: nil)
           path = "#{tickets_path}/password-change"
           request_params = {
-            user_id: user_id,
             result_url: result_url,
-            new_password: new_password,
+            user_id: user_id,
             connection_id: connection_id,
-            email: email
+            email: email,
+            ttl_sec: ttl_sec,
```

Comment:
We probably want to force this to an integer or check if an integer was passed. Better to raise the error here than make a bad API call.